test/buy-tshirt.spec.ts

```diff
@@ -1,38 +1,48 @@
 import { $, browser } from 'protractor';
+import { MenuContentPage, ProductListPage, ProductAddedModalPage, OrderSummaryPage,
+  SignInStepPage, AddresStepPage, ShippingStepPage, PaymentStepPage,
+  BankPaymentPage } from '../src/page';
 
 describe('Buy a t-shirt', () => {
+  const menuContentPage: MenuContentPage = new MenuContentPage();
+  const productListPage: ProductListPage = new ProductListPage();
+  const productAddedModalPage: ProductAddedModalPage = new ProductAddedModalPage();
+  const orderSummaryPage: OrderSummaryPage = new OrderSummaryPage();
+  const signInStepPage: SignInStepPage = new SignInStepPage();
+  const addresStepPage: AddresStepPage = new AddresStepPage();
+  const shippingStepPage: ShippingStepPage = new ShippingStepPage();
+  const paymentStepPage: PaymentStepPage = new PaymentStepPage();
+  const bankPaymentPage: BankPaymentPage = new BankPaymentPage();
   beforeEach(() => {
     jasmine.DEFAULT_TIMEOUT_INTERVAL = 120000;
   });
 
   it('then should be bought a t-shirt', async () => {
     await browser.get('http://automationpractice.com/');
     await(browser.sleep(10000));
-    await $('#block_top_menu > ul > li:nth-child(3) > a').click();
+    await menuContentPage.goToTshirtMenu();
     await(browser.sleep(3000));
-    await $('#center_column a.button.ajax_add_to_cart_button.btn.btn-default').click();
+    await productListPage.addShirtToCart();
     await(browser.sleep(3000));
-    await $('[style*="display: block;"] .button-container > a').click();
+    await productAddedModalPage.proccedToCheckout();
     await(browser.sleep(3000));
-    await $('.cart_navigation span').click();
+    await orderSummaryPage.confirmAndCheckout();
```

Comment:
here, you should use the page object that should implement the `summary-step.page.ts`